h2/src/main/org/h2/api/ErrorCode.java

```diff
@@ -1758,7 +1758,7 @@
      * DROP DOMAIN UNKNOWN;
      * </pre>
      */
-    public static final int USER_DATA_TYPE_NOT_FOUND_1 = 90120;
+    public static final int DOMAIN_NOT_FOUND_1 = 90120;
```

Comment:
when changing existing entries in this class, please provide backwards compat constants with the old names that point to the new names (and are marked deprecated), since this file is part of stable API

Reply:
Oh, I forget that this class is in `api` package. Fixed.

Reply:
`TestScript` was broken by such fix. I'll try to exclude deprecated items in it.